glmmTMB/src/glmmTMB.cpp

```diff
@@ -772,6 +775,19 @@ Type objective_function<Type>::operator() ()
         SIMULATE {
           yobs(i) = glmmtmb::rtweedie(s1, s2, s3);
         }
+      case lognormal_family:
+	// FIXME: documentation??
+	// parameterized in terms of mean and SD on *data* scale, i.e.
+	// mu = exp(logmu + logsd^2/2)
+	// sd = sqrt((exp(logsd^2)-1)*exp(2*logmu + logsd^2)) = mu*sqrt(exp(logsd^2)-1)
+	// 1+(sd/mu)^2 = exp(logsd^2)
+	// logsd = sqrt(log(1+(sd/mu)^2))
+	// logmu = log(mu)- 
+        s1 = log(1 + pow(phi(i)/mu(i), 2.0));
+        s2 = sqrt(s1);
+        s3 = log(mu(i)*mu(i)) - log(mu(i)*mu(i) + phi(i)*phi(i))/Type(2.0); //from Wikipedia
```

Comment:
I don't think I needed to change this. I was just trying everything I could think of. We should consider how to write it to minimize overflow (probably not this way).